packages/docs/modules/page-config/blocks/shared/markdown/MarkdownView.vue

```diff
@@ -109,7 +109,7 @@ const codeBorder = computed(() => {
     position: relative;
 
     &::after {
-      content: "\279A";
+      content: "";
```

Comment:
How about removing whole `::after` element CSS?

Reply:
sure, let me update the PR

Reply:
updated the PR